cloud/ibm/ibm-cloud-iks.md

```diff
@@ -1,186 +1,105 @@
 
-$ ETCDCTL_API=3 etcdctl --endpoints=https://portal-ssl294-1.bmix-wdc-yp-a7a89461-abcc-45e5-84d7-cde68723e30d.588786498.composedb.com:15832,https://portal-ssl275-2.bmix-wdc-yp-a7a89461-abcc-45e5-84d7-cde68723e30d.588786498.composedb.com:15832 --user=root:XXXXXXXXXXXXXXX member list -w table
-+------------------+---------+------------------------------------------+-------------------------+-------------------------+
-|        ID        | STATUS  |                   NAME                   |       PEER ADDRS        |      CLIENT ADDRS       |
-+------------------+---------+------------------------------------------+-------------------------+-------------------------+
-|  418ef52006c5049 | started | etcd129.sl-us-wdc-1-memory.2.dblayer.com | http://10.96.146.3:2380 | http://10.96.146.3:2379 |
-|  bc4aa29071d56be | started | etcd113.sl-us-wdc-1-memory.0.dblayer.com | http://10.96.146.4:2380 | http://10.96.146.4:2379 |
-| a52c701de81e1e64 | started | etcd113.sl-us-wdc-1-memory.1.dblayer.com | http://10.96.146.2:2380 | http://10.96.146.2:2379 |
-+------------------+---------+------------------------------------------+-------------------------+-------------------------+
-```
+**How can I make sure that my data is stored highly available?** </br>
+You need at least 3 worker nodes in your Portworx cluster so that Portworx can replicate your data across nodes. By replicating your data across worker nodes, Portworx can ensure that your stateful app can be rescheduled to a different worker node in case of a failure without losing data. For even higher availability, use a [multizone cluster](https://console.bluemix.net/docs/containers/cs_clusters_planning.html#multizone) and replicate your volumes across SDS worker nodes in 3 or more zones.
 
-Please make note of both of the `--etcd-endpoints` as well as the `--user=root:PASSWORD` string 
+## Step 2: Creating or preparing your cluster for Portworx
+{: #cluster-create}
 
-## Deploy Portworx via Helm Chart
+To install Portworx, you must have an IBM Cloud Kubernetes Service cluster that runs Kubernetes version 1.10 or higher. To make sure that your cluster is set up with worker nodes that offer best performance for you Portworx cluster, review [Step 1: Choosing the right worker node flavor for your IBM Cloud Kubernetes Service cluster to support Portworx](#worker-flavor). 
 
-Make sure you have installed `helm` and run `helm init`
+To create or prepare your cluster for Portworx: 
 
-Follow these instructions to [Deploy Portworx via Helm](https://github.com/portworx/helm/blob/master/charts/portworx/README.md)
+1. [Install the IBM Cloud CLI](https://console.bluemix.net/docs/cli/index.html#overview) to create and manage your Kubernetes clusters in IBM Cloud Kubernetes Service. The IBM Cloud CLI includes the latest version of Docker, Helm, Git, and the `kubectl` CLI so that you do not need to install these packages separately. 
```

Comment:
Do they need to install the `ks` plug-in? Maybe just say `Install the IBM Cloud CLI and Kubernetes Service plug-in`

Reply:
I chose the IBM Cloud CLI package as they need IKS, helm, git, kubectl.